src/styles/components/home-page.scss

```
@@ -1,134 +1,108 @@
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  flex: 1;
+}
+
 .homepage{
+  height: 100%;
   font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Ubuntu,sans-serif;
-  flex: 1;
   display: flex;
   flex-direction: column;
+  align-items: center;
+  backdrop-filter: blur(8px);
   justify-content: center;
-  background-image: url('/images/homepage.jpg');
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: cover;
   
+  &__header{
+    text-align: center;
+  }
 
-  &__wrapper{
+  &__search-wrapper{
+    width: calc(100vw - 50%);
+    height: 200px;
+    background-color: $background-default;
     display: flex;
     flex-direction: column;
-    align-items: center;
-    backdrop-filter: blur(8px);
     justify-content: center;
-    flex: 1;
-    
-  }
+    align-items: center;
+    box-shadow: $box-shadow-default;
+    border-radius: $br-sm;
 
+    @media only screen and (max-width: 599px){
+      width: 90%;
+      padding: $m-base 0px;
+    }
+  }
+}
   
-  &__heading{
-    text-align: center;
+.search-label{
+  width: 60%;
+  font-size: $font-size-xxl;
+  margin-bottom: $m-xs;
+  padding-left: $m-sm;
+  color: $palette-light;
+
+  @media only screen and (max-width: 599px){
+    padding-left: 0px;
   }
 }
+  
 
-.search-wrapper{
-  width: calc(100vw - 50%);
-  height: 200px;
-  background-color: whitesmoke;
+.search-form{
+  width: 60%;
+  height: 44px;
   display: flex;
-  flex-direction: column;
-  justify-content: center;
+  justify-content: space-between;
   align-items: center;
-  box-shadow: rgb(60 66 87 / 12%) 0px 7px 14px 0px, rgb(0 0 0 / 12%) 0px 3px 6px 0px;
-  border-radius: 8px;
+  box-sizing: border-box;
 
-  &--mobile{
-    width: 90%;
-    padding: 24px 0px;
-  }
-  
-  &__search-label{
-    width: 60%;
-    font-size: 32px;
-    margin-bottom: 8px;
-    padding-left: 16px;
-    color: #9BA6DB;
-
-    &--mobile{
-      padding-left: 0px;
-    }
+  @media only screen and (max-width: 599px){
+    width: auto;
+    flex-direction: column;
+    flex: 1;
+    gap: $m-sm;
   }
-  
 
-  .search-form{
-    width: 60%;
-    height: 44px;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
+  &__search-input{
+    width: 70%;
+    height: 24px;        
     box-sizing: border-box;
+    width: 100%;
+    height: 100%;
+    font-size: $font-size-md;
+    line-height: $font-size-xl;
+    padding: 0px $m-sm;
+    border: unset;
+    border-radius: 4px;
+    outline-color: $palette-dark;
+    background-color: $background-default;
+    box-shadow: rgba(0, 0, 0, 0) 0px 0px 0px 0px, 
+                rgba(0, 0, 0, 0) 0px 0px 0px 0px, 
+                rgba(0, 0, 0, 0) 0px 0px 0px 0px, 
+                rgba(60, 66, 87, 0.16) 0px 0px 0px 1px, 
+                rgba(0, 0, 0, 0) 0px 0px 0px 0px, 
+                rgba(0, 0, 0, 0) 0px 0px 0px 0px, 
+                rgba(0, 0, 0, 0) 0px 0px 0px 0px;
```

This is the part that works:
```suggestion
    box-shadow: rgba(60, 66, 87, 0.16) 0 0 0 1px;
```